src/material/slider/slider-input.ts

```diff
@@ -324,6 +324,12 @@ export class MatSliderThumb implements _MatSliderThumb, OnDestroy, ControlValueA
     if (this._isActive) {
       this._updateThumbUIByValue({withAnimation: true});
     }
+    // special case to handle VoiceOver interacting with
+    // the slider since _onInput does not get called.
+    else if (this._isFocused) {
+      this._updateThumbUIByValue({withAnimation: true});
+      this._slider._onValueChange(this);
```

Comment:
Is there a danger of this being triggered by normal keyboard interaction with the slider and causing double events?

Reply:
I checked and it does cause double events. Im not too sure aside from doing a special check using `cdk/platform` like [tooltip does](https://github.com/angular/components/blob/main/src/material/tooltip/tooltip.ts#L795) to help reduce it. As far as I know there isn't a way to detect if VoiceOver is being used.

Reply:
I'd be curious to see what kind of events are actually getting fired during this interaction, but I don't have an iOS device. I'll see if I can borrow one from somebody. For now at least I don't think we want to land this change as it is.

Reply:
Is there any way to detect that the event is originating from voiceover?